tests/base/model_template.py

```diff
@@ -64,9 +64,7 @@ def __init__(
         self.b1 = b1
         self.b2 = b2
 
-        # if you specify an example input, the summary will show input/output for each layer
-        # TODO: to be fixed in #1773
-        # self.example_input_array = torch.rand(5, 28 * 28)
+        self.example_input_array = torch.rand(5, 28 * 28)
```

Comment:
I thought we were talking about having it as property, right? @awaelchli

Reply:
you mean the user overrides the property function?

Reply:
yes, at least we talked about it but maybe there was some picking issue?